src/agent/onefuzz-supervisor/src/worker/tests.rs

```diff
@@ -67,6 +67,22 @@ pub struct ChildDouble {
     killed: bool,
 }
 
+impl Default for ChildDouble {
+    fn default() -> Self {
+        Self {
+            id: 0,
+            exit_status: Some(ExitStatus {
```

Comment:
This should be an impossible value for `exit_status`: either `code` or `signal` is always set.